ladybug/src/main/resources/springIbisTestTool.xml

```diff
@@ -78,7 +78,7 @@
 	stream can be read after the pipeline has finished (e.g. when Base64Pipe is the last pipe Ladybug would show an
 	empty string while test a pipeline would show the correct result).
 	-->
-	<bean name="closeReportsTask" class="nl.nn.testtool.CloseReportsTask">
+	<bean name="closeReportsTask" class="nl.nn.testtool.CloseReportsTask" autowire="byName">
```

Comment:
Zijn er meerdere testtools? Dit kan toch niet de oplossing zijn?

Reply:
Hoezo meerdere? De setTestTool() heeft geen @Autowired (heb ik nu wel toegevoegd zodat binnenkort in master de  autowire="byName niet meer nodig is als die de nieuwste Ladybug versie gebruikt). Tot die tijd en op de 7.9 branch zorgt  autowire="byName" er voor dat Spring de setTestTool() aanroept.

BTW ik ben alle autowire="byName" in de spring xml langsgelopen en waar nodig @Autowired toegevoegd zodat na Ladybug bump in principe bij alle beans de autowire="byName" weg kan

Reply:
Ah, ja als die annotatie er niet staat snap ik hem!